source/data-api/authenticate.txt

```diff
@@ -0,0 +1,161 @@
+.. _data-api-authenticate-requests:
+
+==============================
+Authenticate Data API Requests
+==============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+Data API endpoints generally require that incoming requests include
+authentication information for the user calling the endpoint. This lets
+the endpoint enforce rules and validate document schemas for each
+request.
+
+Requests must include authentication data in specific request headers.
+App Services uses the following process to authenticate a given request:
+
+1. Check for an ``Authorization`` header. If it's present, try to use
```

Comment:
[optional] it could be worth mentioning somewhere that if the `Authorization: Bearer <token>` header exists but the token validation fails, the entire request fails.

i.e. we do not then attempt to authenticate with credential headers

leaving this as optional though if we feel like this might already be implied